backend/utils/lang_code_to_lang_name.json

```diff
@@ -0,0 +1,30 @@
+{

```

Comment:
Sorry, but where is this file being used? I don't see any refs to this.

Reply:
Same comment for `backend/utils/lang_name_to_lang_code.json` as well

Reply:
Ah, this was an earlier method that I was using but I went with the utils method then. I will remove these.